gitleaves/reports.py

```diff
@@ -68,16 +72,29 @@ def get_next_leaves_by_month(bydates):
     return nextbymonths
 
 
-def export_month_csv(year, month):
-    leaves_csv_path = leaves_csv_path_pat.format(YYYY=year)
-    data = load_csv(leaves_csv_path)
-    outfile = f'/tmp/leaves.{year}.{month}.csv'
-    outdata = ((d.isoformat(), *names)
-               for d, names in
-               data['bymonths'][month].items())
-    csv.writer(open(outfile, 'w')).writerows(outdata)
-    print(f'Data exported to {outfile}')
+# Shekhar’s original in case Jason messes up, — mjb 12/20/2021
+# def export_month_csv(year, month):
+#     leaves_csv_path = leaves_csv_path_pat.format(YYYY=year)
+#     data = load_csv(leaves_csv_path)
+#     outfile = f'/tmp/leaves.{year}.{month}.csv'
+#     outdata = ((d.isoformat(), *names)
+#                for d, names in
+#                data['bymonths'][month].items())
+#     csv.writer(open(outfile, 'w')).writerows(outdata)
+#     print(f'Data exported to {outfile}')
+
+
+def export_month_csv(some_group_of_people):
+    people_by_month = some_group_of_people['group_of_people_by_month']
+    template = templates.get_template('ghwiki/Monthwise.md')
+    if not os.path.exists(ghwiki_reports_dir):
+        os.makedirs(ghwiki_reports_dir)
```

Comment:
@jason-at-scroll  Code to ensure ghwiki_reports_dir exists is repeated a few time in the module. I suggest moving this check just after the paths declaration (currently line 21).

Reply:
<img src="https://www.clipartmax.com/png/small/134-1343436_homer-simpson-homer-simpson-doh-png.png" alt="Homer Simpson - - Homer Simpson Doh Png @clipartmax.com">

Reply:
I need to be more proactive and less apprehensive about changes like this.